96.md

```diff
@@ -84,46 +82,43 @@ it must use the "api_url" field instead.
-
 The `filename` embedded in the file may not be honored by the `server`, which could internally store just the SHA-256 hash value as the file name, ignoring extra metadata.
-The hash is enough to uniquely identify a file, that's why it will be used on the "download" and "delete" routes.
-
-The `server` MUST link the user's `pubkey` string (which is embedded in the decoded header value) as the owner of the file so to later allow them to delete the file.
-Note that if a file with the same hash of a previously received file (so the same file) is uploaded by another user, the server doesn't need to store the new file.
-It should just add the new user's `pubkey` to the list of the owners of the already stored file with said hash (if it wants to save space by keeping just one copy of the same file, because multiple uploads of the same file results in the same file hash).
-
-The `server` MAY also store the `Authorization` header/field value (decoded or not) for accountability purpose as this proves that the user with the unique pubkey did ask for the upload of the file with a specific hash. However, storing the pubkey is sufficient to establish ownership.
+The hash is enough to uniquely identify a file, that's why it will be used on the `download` and `delete` routes.
 
-The `server` MUST reject with 413 Payload Too Large if file size exceeds limits.
+The `server` MUST link the user's `pubkey` string as the owner of the file so to later allow them to delete the file.
 
-The `server` MUST reject with 400 Bad Request status if some fields are invalid.
+### Response codes
 
-The `server` MUST reply to the upload with 200 OK status if the `payload` tag value contains an already used SHA-256 hash (if file is already owned by the same pubkey) or reject the upload with 403 Forbidden status if it isn't the same of the received file.
-
-The `server` MAY reject the upload with 402 Payment Required status if the user has a pending payment (Payment flow is not strictly required. Server owners decide if the storage is free or not. Monetization schemes may be added later to correlated NIPs.).
-
-On successful uploads the `server` MUST reply with **201 Created** HTTP status code or **202 Accepted** if a `processing_url` field is added
-to the response so that the `client` can follow the processing status (see [Delayed Processing](#delayed-processing) section).
+- `200 OK`: File upload exists, but is successful (Existing hash)
+- `201 Created`: File upload successful (New hash)
+- `202 Accepted`: File upload is awaiting processing, see [Delayed Processing](#delayed-processing) section
+- `413 Payload Too Large`: File size exceeds limit
+- `400 Bad Request`: Form data is invalid or not supported.
+- `403 Forbidden`: User is not allowed to upload or the uploaded file hash didnt match the hash included in the `Authorization` header `payload` tag.
+- `402 Payment Required`: Payment is required by the server, **this flow is undefined**.
 
```

Comment:
Contd "no transformations":

+- `422 Unprocessable Content`: An option "no_transform=true" was specified, but server rejected, response body may contain an explanation.

Reply:
Would it make sense to have a `no_transform` field in the `plans` to let the client know beforehand?

Reply:
I believe the 400 response should cover this